config/initializers/visitor.rb

```diff
@@ -21,8 +21,8 @@ class Dot < Arel::Visitors::Visitor
       alias :visit_Integer :visit_String
     end
 
-    class ToSql < Arel::Visitors::Visitor
-      alias :visit_Integer :literal
-    end
+#   class ToSql < Arel::Visitors::Visitor
+#     alias :visit_Integer :literal
+#   end
   end
```

Comment:
I've commented this out for now because of this error
`TypeError: superclass mismatch for class ToSql
/home/k/proj/mapknitter/config/initializers/visitor.rb:24:in `<module:Visitors>'
/home/k/proj/mapknitter/config/initializers/visitor.rb:15:in `<module:Arel>'
/home/k/proj/mapknitter/config/initializers/visitor.rb:14:in `<top (required)>'
/home/k/.rvm/gems/ruby-2.4.4/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'
`
@sashadev-sky You know more about this patch. Can you help me out with this?
Thanks

Reply:
Yeah I actually put this patch to make Ruby 2.4.4 compatible with Rails 3.2. I got that Rails 6 should be where it's removed just from looking at the Rails repo and when they integrated Arel but it could need to go earlier. I would also check out @alexalvas [fork](https://github.com/alaxalves/mapknitter/tree/rails4/) because he said he had it working for 4.2.8

Reply:
@sashadev-sky All I did was an experimental upgrade to see if the problem with `rake assets:precompile` was going to be fixed, since I identified along with @jywarren and @icarito that we a had a sort of big problem with dependencies versions mismatch. I did not move forward with my upgrade because it would be a huge patch regarding a small issue as @jywarren recommended [here](https://github.com/publiclab/mapknitter/pull/363#issuecomment-470319576) and also because that was part of the workflow proposed [here](https://publiclab.org/wiki/gsoc-ideas#MapKnitter+Rails+5+Upgrades).

Reply:
@alaxalves Is correct. I'm having a really hard time finding out the bugs with low test coverage

Reply:
@alaxalves @kaustubh-nair ok makes sense! so @kaustubh-nair is the next step your PR #467 getting merged and then taking it from there? I can add that issue as a dependency to this issue  if so

Reply:
https://github.com/publiclab/mapknitter/pull/467 is part of https://github.com/publiclab/mapknitter/issues/304
We should try to get https://github.com/publiclab/mapknitter/issues/304 done before upgrading Rails.

Reply:
@kaustubh-nair ok got it! Thank you for updating me. I added them as dependencies to this issue

Reply:
Hello @kaustubh-nair, how do you feel about out test coverage so far? I am still adding some tests on #563 but how do you think we could go ahead with this?  And if so is it okay if I pull this locally and take a look? Thanks

Reply:
Hey @cesswairimu tests look awesome now!
I kinda stopped work here because of low test coverage. But I think we can continue now. I'll atleast finish the config changes and you can help out after that, if you want to, as well. 
Thanks!

Reply:
Sounds great, thanks

Reply:
Should I remove the test coverage issues as "blocking" issues now? 
@cesswairimu @kaustubh-nair

Reply:
Solved the merge conflicts